pkg/auto/bms/logic.go

```diff
@@ -126,14 +159,14 @@ func analyseOccupancy(now time.Time, state *ReadState) (occupied, total, noRespo
 }
 
 // analyseTimeOfDay works out if we are currently within the deadband period, and when that might change.
```

Comment:
Not just used for deadband processing any more, update comment to reflect more generic use